frontend/src/components/Admin/FeaturedProductForm/FeaturedProductForm.js

```diff
@@ -0,0 +1,55 @@
+import { useRef } from "react"
+import { useFeaturedProduct } from "../../../hooks/featured-product-hooks"
+import { useMutate } from "../../../hooks/use-mutate"
+import { useParams } from "react-router"
+import FormSkeleton from "../Skeletons/FormSkeleton/FormSkeleton"
+
+const FeaturedProductForm = () => {
+  const formRef = useRef()
+  const { id } = useParams()
+  const { updateFeaturedProduct } = useMutate()
+  const { addFeaturedProduct } = useMutate()
+  let featuredProduct = useFeaturedProduct({ "id": id });
+
+  if (featuredProduct.isError) return <h1>Something went wrong!</h1>
+  if (featuredProduct.isLoading) return <FormSkeleton count={1}/>
+
+  const handleSubmit = async (e) => {
+    e.preventDefault()
+    const formData = new FormData(formRef.current)
+    const data = Object.fromEntries(formData)
+    if (!id) {
+      const input = {
+        productId: parseFloat(data.productId),
+      }
+      addFeaturedProduct.mutate(input)
+    } else {
+      const input = {
+        id: id,
+        productId: parseFloat(data.productId)
+      }
+      updateFeaturedProduct.mutate(input)
+    }
+  }
+
+  return ( 
+    <>
+      <form ref={formRef} onSubmit={handleSubmit} className="max-w-2xl m-6">
+        <h1 className="text-2xl text-center mb-3">{id ? "Edit Featured Product" : "Add Featured Product"}</h1>
+        <label className="admin-form-label">
+          Product:
+          <input
+            className="form-input admin-form-input"
```

Comment:
Class 'form-input' doesn't do anything here, so it can be removed. It should have been removed as well in the ProductForm.js component too, but was missed!

Reply:
Also, is this the correct path for this form? frontend/src/components/Admin/FeaturedProductForm/FeaturedProductForm.js
The reason I'm asking is that the TagForm as well as the ProductForm are located outside this Admin component folder. So should those be moved into Admin folder or this moved out?

Reply:
Yeah I think I want to try sorting them by admin and user app components so its more easily distinguishable. The others would have to be moved too but that'll be a different project on it's own